ndmg/stats/qa_skullstrip.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env python
+
+# Copyright 2016 NeuroData (http://neurodata.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# Created by Vikram Chandrashekhar.
+# Edited by Greg Kiar, Eric Bridgeford and Chuankai Luo.
+# Email: Greg Kiar @ gkiar@jhu.edu   cluo16@jhu.edu
+
+import warnings
+
+warnings.simplefilter("ignore")
+import os
+import re
+import sys
+import numpy as np
+import nibabel as nb
+# import ndmg.utils as mgu
+from argparse import ArgumentParser
+from scipy import ndimage
+from matplotlib.colors import LinearSegmentedColormap
+import matplotlib as mpl
+
+mpl.use("Agg")  # very important above pyplot import
+# from nilearn.plotting.edge_detect import _edge_map as edge_map
+import matplotlib.pyplot as plt
+from ndmg.utils.qa_utils import get_min_max, opaque_colorscale,pad_im
+from ndmg.utils.gen_utils import get_filename, get_braindata
+
+
+
+
+
+def gen_overlay_pngs(
+    brain, original, outdir, loc=0, mean=False, minthr=2, maxthr=95, edge=False):
+    """
+    generate the image combine the brain and original data
+
+    Parameters
+    ----------
+    brain:
+        is the skull striped result, only left the brain
+    original:
+        is the original nii.gz file
+    outdir:
+        directory where output png file is saved
+    loc and mean is to deal with 4d data
+    minthr, maxthr, edge are the variables about the color
+    """
+    try:
+        original_name = get_filename(original)
+    except ValueError:
+        original_name = 'stripped_brain_skull_comparison'
+    brain_data = nb.load(brain).get_data()
+    if brain_data.ndim == 4:  # 4d data, so we need to reduce a dimension
+        if mean:
+            brain_data = brain_data.mean(axis=3)
+        else:
+            brain_data = brain_data[:, :, :, loc]
+    else:  # dim=3
+        brain_data = brain_data
```

Comment:
you can just delete the `else`, it doesn't do anything

Reply:
fixed it in the latest version